### ISSUE TITLE
training code

### ISSUE THREAD

**Issue by javismiles** [MarcoForte/FBA_Matting#28]
good day,
has the training code been released for this model? 
I would like to train it on a new dataset,
is the model receiving as training input pairs of images and ground truth alpha masks?
thank you for any tips
best
